go.sum

```diff
@@ -1,8 +1,1315 @@
 bou.ke/monkey v1.0.2 h1:kWcnsrCNUatbxncxR/ThdYqbytgOIArtYWqcQLQzKLI=
 bou.ke/monkey v1.0.2/go.mod h1:OqickVX3tNx6t33n1xvtTtu85YN5s6cKwVug+oHMaIA=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-cloud.google.com/go v0.38.0 h1:ROfEUZz+Gh5pa62DJWXSaonyu3StP6EA6lPEXPI6mCo=
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
+cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6AU=
```

Comment:
so much versions of a same pkg. do you run `go mod tidy`?

Reply:
sure :)  that is a necessary action.
All module in used.
```shell
➜  erda-infra git:(feat/support-go1.22) ✗ go mod tidy -v
➜  erda-infra git:(feat/support-go1.22) ✗ 
```
I try to remove `go.sum` and re-created, nothing changed within module `cloud.google.com/go`.

Reply:
It seems that now `go.sum` changed with re-create, PTAL.